eslint.config.js

```diff
@@ -1,6 +1,5 @@
 // @ts-check
 const emotionPlugin = require('@emotion/eslint-plugin');
-const { fixupPluginRules } = require('@eslint/compat');
```

Comment:
The testing-library plugin was [updated to 6.4.0 recently](https://github.com/grafana/grafana/pull/96391) which includes the fixed config, so we don't need to run it through compat anymore